src/workspace/vite.config.ts

```diff
@@ -12,6 +12,13 @@ function getBaseUrl(dev: boolean) {
 const copyOpenfinPlugin = (isDev: boolean): Plugin[] => {
   const env = process.env.ENVIRONMENT || "local"
 
+  const reactiveAnalyticsUrl =
+    env === "local"
```

Comment:
bit fiddly, but need the local address to be there for local "development" and "production" builds
the usual dance around prod -> demo makes the rest of it unnecessarily messy ..